src/testament.janet

```diff
@@ -611,3 +612,11 @@
   (set reports @{})
   (set print-reports nil)
   (set on-result-hook (fn [&])))
+
+(defmacro defsuite! [& body]
+  ```
+  Automatically runs the tests defined in the macro body
+  ```
+  ~(do
+     ,;body
+     (,(comptime run-tests!))))
```

Comment:
I should have asked this before so my apologies. My macro-fu is pretty weak, why are you using `comptime` here?

Reply:
So that was intended for keeping other functions from defining their own `run-tests!` that conflicts with the one in testament. 

It's not strictly necessary.

Reply:
@yumaikas That sounds like a good idea but I confess I was too much of a dummy to get it to work with my changes that added `do-tests` so I took it out. Happy to have it added back.

Reply:
If it's hard to get working, then let's not keep it around.